report/src/sections/01-introduction.tex

```diff
@@ -1,22 +1,29 @@
-In this paper we use static program analysis, specifically abstract interpretation.
+\IEEEPARstart{S}{oftware} analysis is a broad field encompassing various techniques and methodologies, each designed to address specific aspects of software systems.
+Among these techniques, data flow analysis is a fundamental method for systematically analyzing software behaviour.
+Abstract interpretation is a powerful tool for examining software behaviour through static analysis, providing a formal framework for reasoning about program properties and the data flow of a program~\cite{cousot_abstract_1977}.
 
-Abstract interpretation, first proposed by Cousot and Cousot in 1977~\cite{cousot_abstract_1977}, has emerged as a fundamental method for static program analysis.
-Over the years, it has evolved into a versatile tool, finding applications across various programming paradigms and system architectures.
-Its utility extends beyond traditional imperative programming to encompass object-oriented designs and concurrent systems~\cite{gustafsson_analyzing_2013, mine_static_2023}.
-Abstract interpretation has played a pivotal role in analyzing a spectrum of program properties, from basic safety and liveness concerns to intricate security considerations~\cite{mastroeni_abstract_2011}.
+Static program analysis, mainly through abstract interpretation, is crucial for finding information that is guaranteed to hold across all executions of a program~\cite{jackson_software_2000}.
 
-Building upon this, Halder and Cortesi have expanded the application of abstract interpretation to domains like query languages~\cite{halder_abstract_2012}.
-This paper continues in these footsteps, aiming to further explore and extend the capabilities of abstract interpretation within the context of query languages.
+
+This paper leverages abstract interpretation to analyze software behaviour.
+Abstract interpretation is a mathematical framework that facilitates reasoning about the correctness of static analysis.
+
+Abstract interpretation, first introduced by Cousot and Cousot in 1977~\cite{cousot_abstract_1977}, has become a foundational method for static program analysis.
+Over the years, it has evolved into a versatile tool, applicable across various programming paradigms and system architectures.
+Its utility extends beyond traditional imperative programming, including object-oriented designs and concurrent systems~\cite{gustafsson_analyzing_2013, mine_static_2023}.
+
+Abstract interpretation has been instrumental in analyzing various program properties, from basic safety and liveness concerns to complex security considerations~\cite{mastroeni_abstract_2011}.
+
+Building on this foundation, Halder and Cortesi have extended the application of abstract interpretation to domains such as query languages~\cite{halder_abstract_2012}.
+This paper aims to further explore and expand the capabilities of abstract interpretation within the context of query languages.
```

Comment:
Jeg er ked af at sige det men introduktionen var bedre før. Denne introduktion er et maltrakteret clusterfuck.
Overvej at rette Gabrielas kommentar frem for at omskrevet halvdelen.